cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -27,8 +26,20 @@ <h2 class="header-slug">
         </span>
     </h2>
     {% endif %}
-    {#% TODO: Add condition to display various related post types %#}
-    {{ _related_posts_list() }}
+    {% set related_post_types = page.related_posts %}
+    {% if related_post_types | length  > 1 %}
```

Comment:
That is what is in the spec. @ajbush can you provide some guidance on this issue?